### PR TITLE
Add iCount QC status process

### PR DESCRIPTION
**Pull request by jkokosar**

### CODE REVIEW

File: resolwe_bio/tools/icount_qc.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    return status, message
+
+
+def main():
+    """Invoke when run directly as a program."""
+    args = parse_arguments()
+
+    with open(args.mapping_stats) as stats:
+        for line in stats:
```
Comment (JureZmrzlikar) [inline resolwe_bio/tools/icount_qc.py]: It is more readable to split the line in the beginning, once for all.

File: resolwe_bio/tools/icount_qc.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Run iCount demultiplexing.")
+    parser.add_argument('mapping_stats', help="Star mapping stats.")
+    return parser.parse_args()
+
+
+def report_qc(cdna, reads):
```
Comment (JureZmrzlikar) [inline resolwe_bio/tools/icount_qc.py]: I would avoid naming this quantity cDNA. cDNA scores for each position are computed in a complex process (`iCount xlsites`). cDNA is not equal to the sum of `umr + mlr`.

**Comment by jkokosar** [comment]
@JureZmrzlikar @mstajdohar this PR has been refactored. An additional commit is added. Please re-review.

File: resolwe_bio/processes/icount/xlsites.yml [hunk 3]
```diff
@@ -81,6 +84,9 @@
 
       re-checkrc
 
+      READS=`samtools view -F 0x04 -c {{bam.bam.file}}`
+      icount_qc.py $READS $(gzip -d -c $MULTINAME | cat | wc -l)
```
Comment (JureZmrzlikar) [inline resolwe_bio/processes/icount/xlsites.yml]: You could use zcat.

Comment (JureZmrzlikar) [inline resolwe_bio/processes/icount/xlsites.yml]: You could use zcat.

**Comment by mstajdohar** [comment]
Included in #272

**Pull request closed without merging**